homework-submission/week1/src/components/RemoveTodo.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+class RemoveTodo extends Component {
+    render(){
+        return(            
+                <button onClick= {()=>{this.props.remove(this.props.i)}} type='submit'>Remove</button>
```

Comment:
No need for `type="submit"` here, because this button is not a part of a `<form>`.